.gitignore

```diff
@@ -104,3 +105,8 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+$HOME/.pyenv/pyenv_win-2.64.11.dist-info/METADATA
```

Comment:
can you revert the changes to .gitignore? these are only applicable  to your setup